### PR TITLE
Fix latest channel package query

### PR DESCRIPTION
**Pull request by chefsalim**
This change fixes the latest channel package query to correctly return the latest package version. This is the same change that was made for the package latest query, now expanded to channels.

Signed-off-by: Salim Alam <salam@chef.io>

giphy 34

### CODE REVIEW

File: components/builder-originsrv/src/migrations/origin_channels.rs [hunk 1]
```diff
@@ -130,9 +130,7 @@ pub fn migrate(migrator: &mut Migrator) -> Result<()> {
                           WHERE o.name = op_origin
                           AND oc.name = op_channel
                           AND op.ident LIKE (op_ident  || '%')
-                          AND op.target = op_target
-                          ORDER BY op.ident DESC
-                          LIMIT 1;
+                          AND op.target = op_target;
```
Comment (reset) [inline components/builder-originsrv/src/migrations/origin_channels.rs]: Will we need to manually run this function update on live or did we do this the other night/morning?

Comment (chefsalim) [inline components/builder-originsrv/src/migrations/origin_channels.rs]: Yeah we'll need to manually run it.

Comment (mwrock) [inline components/builder-originsrv/src/migrations/origin_channels.rs]: I think we should make it a v2 in a new migration. yeah? Seems like manually running ad hoc alter queries is not something we want to do outside of an emergency (if then at all).

Comment (chefsalim) [inline components/builder-originsrv/src/migrations/origin_channels.rs]: Good idea - will update.

**Comment by reset** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/236849120).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/236849120).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**